app/db/database.py

```diff
@@ -8,7 +8,7 @@
 DB_STRING = 'postgres://{}:{}@{}:{}/{}'.format(
     config.get('POSTGRES_USER', 'wps'),
     config.get('POSTGRES_PASSWORD', 'wps'),
-    config.get('POSTGRES_HOST', 'db'),
+    config.get('POSTGRES_HOST', 'localhost'),
```

Comment:
This will become `config.get('POSTGRES_HOST', 'db'),` once you pull the main branch (merged with Sy's)